services/users/project/db/extensions.sh

```diff
@@ -0,0 +1,3 @@
+for DB in $(psql -t -c "SELECT datname FROM pg_database WHERE datname NOT IN ('postgres', 'template0', 'template1')"); do
+  psql -d $DB -c "CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";"
```

Comment:
UUID types are not supported by default, the following script adds the extension to all development databases.